### PR TITLE
🌈 Add colours and graph to the doc 🌈

### PR DESCRIPTION
**Pull request by nonifier**
Because life is always so much better when it's colourful.

I you want to have a wee-look at the result : https://github.com/nonifier/navitia/blob/raptor_rainbow_doc/documentation/rfc/doc-routing-algorithm.md

### CODE REVIEW

File: documentation/rfc/doc-routing-algorithm.md [hunk 1]
```diff
@@ -81,9 +84,9 @@ Starting from A at 7:45:
 |level| A  | B  | C  | D  | E  | F  |  G      |
 |-----|----|----|----|----|----|----|---------|
 |TR0  |7:45|    |    |    |    |    |         |
-|PT1  |    |8:30|8:33|8:36|8:50|    |         |
+|PT1  |    |8:30 ![](https://placehold.it/15/010101/ffffff?text=2+) |8:33 ![](https://placehold.it/15/010101/ffffff?text=2+)  |8:3 6![](https://placehold.it/15/010101/ffffff?text=2+)  |8:50 ![](https://placehold.it/15/ff9b19/000000?text=1+) |    |         |
```
Comment (pbougue) [inline documentation/rfc/doc-routing-algorithm.md]: extra space at 8:36

**Comment by nonifier** [comment]
[skip-ci]

**Review by pbougue — APPROVED** [review]
:sparkles: :heart_eyes: :trophy:
Really helps reading it, too :+1:

File: documentation/rfc/doc-routing-algorithm.md (hunk 1 above)
Comment (nonifier) [reply documentation/rfc/doc-routing-algorithm.md]: :+1:

**Review by benoit-bst — APPROVED** [review]
Unicorn job :unicorn:
I like it

**Review by kinnou02 — APPROVED** [review]

**Pull request merged**